Makefile

```diff
@@ -1,13 +1,38 @@
 VERBOSE ?= @
 TEST_PROCS ?= $(shell nproc)
+ORIGIN ?= https://github.com/Componolit
```

Comment:
Maybe we should use a more specific variable name (`RECORDFLUX_ORIGIN`?) to reduce the chance of accidental clashes?

Reply:
Changed.